graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/Message.java

```diff
@@ -65,7 +66,7 @@
     public static final String FIELD_LEVEL = "level";
     public static final String FIELD_STREAMS = "streams";
 
-    private static final Pattern VALID_KEY_CHARS = Pattern.compile("^[\\w\\.\\-@]*$");
+    private static final Pattern VALID_KEY_CHARS = Pattern.compile("^[\\w\\.\\-@/]*$");
```

Comment:
The question is rather which characters we want to allow additionally. In the end, this pattern controls which replacement characters can be used in the first place.